src/app/forms.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, *args, **kwargs):
     sourceUrl = forms.CharField(label='Source / URL', required=False)
     osiApproved = forms.CharField(label="OSI Status", widget=forms.Select(choices=OSI_CHOICES))
     exampleUrl = forms.CharField(label='Example Projects / URL', required=True)
-    comments = forms.CharField(label='Comments', required=False, widget=forms.Textarea(attrs={'rows': 4, 'cols': 40}))
+    comments = forms.CharField(label='Comments', required=True, widget=forms.Textarea(attrs={'rows': 4, 'cols': 40}))
```

Comment:
I checked the `views.py` code for `submitNewLicense` which checks if the form is valid but doesn't throw/show any error to the end user when backend form verification fails. I think we would need to introduce check on UI(javascript) so that this field always gets filled- `checkform` in `submit_new_license.html`